lib/base.js

```diff
@@ -54,6 +59,75 @@ Base.prototype._checkStatusForError = function(statusCode, body) {
     }
 };
 
+Base.prototype.register = function({username, password, fields}, done) {
+    if (!username) {
+        throw new AirtableError('NOT_FOUND', "Missing parameter 'username' required for Base#register", 404);
+    }
+    if (!password) {
+        throw new AirtableError('NOT_FOUND', "Missing parameter 'password' required for Base#register", 404);
+    }
+    if (this._airtable._endpointUrl.includes('https://api.airtable.com')) {
+        throw new AirtableError('NOT_AUTHORIZED', 'Base#register cannot be used with api.airtable.com. Please configure this Airlock client to use an Airlock endpoint URL.', 403);
+    }
+    if (this._user !== null || this._getLocalStorage('user') !== null) {
+        throw new AirtableError('NOT_AUTHORIZED', 'Already logged in');
+    }
+    runAction(this, 'post', '/__DANGEROUSLY__USE__TABLE__TO__LET__USERS__REGISTER', {}, {username, password, fields}, (err, data) => {
+        // this._user = data;
```

Comment:
Are we still removing this? I think based on the conversations we had in Slack, we want to set up the user inside of the registration state as soon as their registration goes through. (See here: https://cal-blueprint.slack.com/archives/CNE3KQWUA/p1573798476139100)

Reply:
good catch! Thanks :)